R/tm_g_gh_density_distribution_plot.R

```diff
@@ -419,11 +420,13 @@ srv_g_density_distribution_plot <- function(id, # nolint
       teal.code::join(create_plot(), create_table())
     })
 
+    code <- reactive(
+      paste0(teal.code::get_code(joined_qenvs()), "\nprint(tbl)")
```

Comment:
(same comment as above)

Why not add `tbl` (without `print()`) on line 394? (instead of pasting it)

That's what we used in `tmg::tm_outliers` ([link here](https://github.com/insightsengineering/teal.modules.general/blob/main/R/tm_outliers.R#L959-L977)), which prints the table when running the reproducible code, but doesn't show on console in a teal app.

If not then avoid pretending `\n` in favor or `sep` argument.
```suggestion
      paste(teal.code::get_code(joined_qenvs()), "print(tbl)", sep = "\n")
```